index.html

```diff
@@ -16,10 +16,15 @@
       <nav class="nav">
         <a href="">Welcome</a>
         <i class="fas fa-bars"></i>
```

Comment:
- [ ] [OPTIONAL]: please notice that in Figma there is no logo on the left side for desktop views 😯 : 

Yours | Figma
---------|---------
![image](https://user-images.githubusercontent.com/77038610/144696949-493c18d0-a562-4f58-b3d8-e33800ba8e31.png)  | ![image](https://user-images.githubusercontent.com/77038610/144696971-ad168a58-4c3f-4b0d-9336-f55ec296b075.png)

Please consider deleting logo from desktop view.